server/testsupport/timelineTestDataBuilder.ts

```diff
@@ -0,0 +1,54 @@
+import type { Timeline } from 'viewModels'
+
+export default function aValidTimelineResponse(): Timeline {
```

Comment:
I think this might need some thought .... is it returning a `TimelineResponse`? (the API type) or a `Timeline`? (the UI view model)
Personally I think we want both, so I'd be tempted to rename the method to `aValidTimeline` (everything else looks OK I think); and then create a 2nd test builder file called `timelineResponseTestDataBuilder.ts` with a method `aValidTimelineResponse` that returns a `TimelineResponse`